packages/did/src/Did.chain.ts

```diff
@@ -101,6 +101,8 @@ export async function queryAllServicesEncoded(
   const encodedEndpoints = await api.query.did.serviceEndpoints.entries<
     Option<IServiceEndpointChainRecordCodec>
   >(didIdentifier)
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
Q: What is typescript complaining about here?

Reply:
https://github.com/KILTprotocol/sdk-js/commit/237a30863f63ce624399146463288a409348658e